js/ui/popup/ui.popup.js

```diff
@@ -280,6 +271,12 @@ const Popup = Overlay.inherit({
         this.callBase();
     },
 
+    _createBodyOverflowManager: function() {
+        if(hasWindow()) {
+            this._bodyOverflowManager = createBodyOverflowManager();
```

Comment:
We can use [null object pattern](https://refactoring.guru/introduce-null-object) to get rid of conditions like this here and other places using `_bodyOverflowManager`.
You can check how it's used in [ResizeObserver implementation](https://github.com/DevExpress/DevExtreme/blob/23_1/js/core/resize_observer.js#L14).

Reply:
[v](https://github.com/DevExpress/DevExtreme/pull/23926/commits/fd2b6b773237f86f0fd2b04d5786a12da38224c0)